src/main/java/ro/chirila/programarispital/repository/dto/TypeOfServiceDTO.java

```diff
@@ -0,0 +1,15 @@
+package ro.chirila.programarispital.repository.dto;
+
+public class TypeOfServiceDTO {
+    private String service;
+
+    //region Getter and setter
+    public String getService() {
+        return service;
+    }
+
+    public void setService(String service) {
```

Comment:
you don't need setters, create a constructor with the required fields. Using `record` classes automatically does this for you.